### PR TITLE
update ordo exclusions

### PR DESCRIPTION
**Pull request by twhetzel**
related to [mondo #7437](https://github.com/monarch-initiative/mondo/issues/7437#issuecomment-2110355523)

## Overview
TODO

### Documentation
Was the documentation added/updated under `docs/`?
- [ ] Yes
- [x] No, updates to the docs were not necessary after careful consideration

### QC
Was the full pipeline run before submitting this PR using `sh run.sh make build-mondo-ingest` on this branch (after 
`docker pull obolibrary/odkfull:dev`), and no errors occurred?
- [ ] Yes
- [x] No, there are no functional (code-related) changes to the pipeline in the PR, so no re-run is necessary
   
### New Packages
Were any new *Python packages* added?
- [ ] Yes, [requirements.txt.full](https://github.com/INCATools/ontology-development-kit/blob/master/requirements.txt.full) was updated and an [ODK issue](https://github.com/INCATools/ontology-development-kit/issues/new) submitted
- [x] No

Were any other *non-Python packages* added?
- [ ] Yes, [Dockerfile](https://github.com/INCATools/ontology-development-kit/blob/master/Dockerfile) updated and an [ODK issue](https://github.com/INCATools/ontology-development-kit/issues/new) submitted
- [x] No

### PR Review and Conversations Resolved
Has the PR been sufficiently reviewed by at least 1 team member of the Mondo Technical team and all threads resolved?
- [ ] Yes

### CODE REVIEW

File: src/ontology/config/ordo_exclusions.tsv [hunk 1]
```diff
@@ -176,4 +176,5 @@ Orphanet:522514	Congenital optic disc excavation of genetic origin	FALSE	MONDO:e
 Orphanet:522562	Genetic superficial corneal dystrophy	FALSE	MONDO:excludeGroupingOriginGenetic
 Orphanet:519337	Disorder with optic nerve compression	FALSE	MONDO:excludeGroupingPhenotype
 Orphanet:522520	Syndromic genetic disorder with strabismus	FALSE	MONDO:excludeGroupingPhenotype
-Orphanet:522530	Rare genetic disorder with entropion	FALSE	MONDO:excludeGroupingPhenotype
\ No newline at end of file
+Orphanet:522530	Rare genetic disorder with entropion	FALSE	MONDO:excludeGroupingPhenotype
+Orphanet:158038	Primary hemophagocytic lymphohistiocytosis	FALSE	MONDO:excludeGrouping
```
Comment (joeflack4) [inline src/ontology/config/ordo_exclusions.tsv:180]: Just giving a face check review. I'm not familiar with how these exclusion codes work, but it does indeed seem that `MONDO:excludeGrouping` matches with what is being asked! (https://github.com/monarch-initiative/mondo/issues/7437)

File: src/ontology/config/ordo_exclusions.tsv
Comment (joeflack4) [inline src/ontology/config/ordo_exclusions.tsv:1]: @twhetzel I added the `mondo-ingest` PR template to the OP, and checked some of the boxes off for you.

And it is a configuration change, not a code change, so I see it is appropriate to merge it to `main`. And so I also take it that no QC build required for this one.

Comment (twhetzel) [reply src/ontology/config/ordo_exclusions.tsv:1]: Yes, that is my understanding looking at Sabrina's past PRs for this type of change.

**Comment by sabrinatoro** [comment]
I am a bit confused by these comments. 

1) All terms excluded should have an exclusion reason. This exclusion code is a way to document "why" the term is excluded from Mondo. (Maybe it is already or will at a later time be included in some mapping files)
2) @twhetzel I think you could use a more specific exclusion code. For example: MONDO:excludeGroupingHeritability (see [exclusion code document](https://mondo.readthedocs.io/en/stable/editors-guide/exclusion-reasons/#exclusion-reasons))
3) This is just a file update, so I don't think we need to do anything else.

**Comment by twhetzel** [comment]
Joe - I am here on this PR as a curator for information for Sabrina to review, but good for you to have looked at this too.

**Review by sabrinatoro — APPROVED** [review]

**Comment by joeflack4** [comment]
> Joe - I am here on this PR as a curator for information for Sabrina to review, but good for you to have looked at this too.

@twhetzel Yes, feel free to overrule, but I intend to at least look at every issue and PR that comes through `mondo-ingest`.

**Pull request merged**